src/com/visualization/piechart/piechart.js

```diff
@@ -0,0 +1,69 @@
+import {h, Component} 				from 'preact/preact';
+
+import PieSegment					from 'com/visualization/piechart/piesegment';
+
+export default class PieChart extends Component {
+
+	constructor( props ) {
+		super(props);
+
+    }
+
+    // Converts an array of number to an array of percentages
+    convertToPercentage ( values ) {
+        // calculate total
+        let total = 0;
+        values.forEach( ( v ) => ( total += v ) );
+
+        // scale values so there sum is 100%
+        let percentages = values.map( ( v ) => ( (100 * v) / total ) );
+
+        return percentages;
+    }
+
+    render( props ) {
+
+        if ( !(props && props['lables'] && props['values'])) {
```

Comment:
Super minor (probably a typo), but prefer
```
if ( !(props && props['lables'] && props['values']) ) {
```
noticing the added space at the end separating the `))`.

Reply:
Added spacing